### PR TITLE
Few DataStructures Trivia Added

### PR DESCRIPTION
**Pull request by Gautam-Hegde**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1,4 +1,40 @@
...
+    Link: "https://www.geeksforgeeks.org/stack-data-structure/",
+  },
+  {
+    Question: "Which of the following is a Divide and Conquer algorithm?",
+    Answer: "Merge Sort",
+    Distractor1: "Selection Sort",
+    Distractor2: "Heap Sort",
+    Distractor3: "Bubble Sort",
+    Explanation: "Merge Sort is a Divide and Conquer algorithm. It divides the input array into two halves, calls itself for the two halves, and then merges the two sorted halves.",
+    Link: "https://www.geeksforgeeks.org/merge-sort/",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:27]: Link:"https://www.freecodecamp.org/news/an-intro-to-advanced-sorting-algorithms-merge-quick-radix-sort-in-javascript-b65842194597/",

File: src/data/full-quiz.js [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
+  },
+  {
+    Question: "Worst case time complexity to access an element in a BST can be?",
+    Answer: "O(n)",
+    Distractor1: "O(n*logn)",
+    Distractor2: "O(n*n)",
+    Distractor3: "O(logn)",
+    Explanation: "We might need to visit all the nodes in the BST,hence the time complexity would be O(n).",
+    Link: "https://www.geeksforgeeks.org/binary-search-tree-set-1-search-and-insertion/",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:36]: Link:"https://www.freecodecamp.org/news/binary-search-tree-what-is-it/",

File: src/data/full-quiz.js [hunk 3]
```diff
@@ -1,4 +1,40 @@
...
+    Question: "Which of the following is a Divide and Conquer algorithm?",
+    Answer: "Merge Sort",
+    Distractor1: "Selection Sort",
+    Distractor2: "Heap Sort",
+    Distractor3: "Bubble Sort",
+    Explanation: "Merge Sort is a Divide and Conquer algorithm. It divides the input array into two halves, calls itself for the two halves, and then merges the two sorted halves.",
+    Link: "https://www.geeksforgeeks.org/merge-sort/",
+  },
+  {
+    Question: "Worst case time complexity to access an element in a BST can be?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:30]: Proposed fix:

What is the worst case time complexity for accessing an element in a BST(Binary Search Tree)?

File: src/data/full-quiz.js [hunk 4]
```diff
@@ -1,4 +1,40 @@
...
+    Answer: "O(n)",
+    Distractor1: "O(n*logn)",
+    Distractor2: "O(n*n)",
+    Distractor3: "O(logn)",
+    Explanation: "We might need to visit all the nodes in the BST,hence the time complexity would be O(n).",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:35]: Proposed fix:
The worst case time complexity would be O(n) because we might need to visit all of the nodes.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @Gautam-Hegde ! 

Thank you for your questions! 
I just have a few requested changes, then we can merge your questions into the site. 👍

File: src/data/full-quiz.js [hunk 5]
```diff
@@ -1,4 +1,40 @@
...
+    Link: "https://www.ibm.com/docs/en/spectrum-protect/8.1.9?topic=parameters-file-specification-syntax",
+  },
+  {
+    Question: "An Example for LIFO structure is",
+    Answer: "Stacks",
+    Distractor1: "Priority Queues",
+    Distractor2: "Trees",
+    Distractor3: "Queues",
+    Explanation: "Stack is a linear data structure which follows a particular order in which the operations are performed. The order may be LIFO(last in first out) or FILO(First In Last Out)",
+    Link: "https://www.geeksforgeeks.org/stack-data-structure/",
```
[the remark this answers is not in the document]
Comment (Gautam-Hegde) [reply src/data/full-quiz.js]: Sorry, I won't repeat that mistake
Implemented all the requested changes ✔

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Comment by jdwilkin4** [comment]
Congrats on your first merged PR into the freeCodeCamp Developer Quiz Site!
We look forward to future contributions 👍

**Pull request merged**